src/main/java/org/folio/linked/data/configuration/ValidatorsConfiguration.java

```diff
@@ -0,0 +1,21 @@
+package org.folio.linked.data.configuration;
+
+import static org.folio.linked.data.util.Constants.Validators.VALIDATOR_CHAIN_LD;
+
+import org.folio.linked.data.domain.dto.ResourceRequestDto;
+import org.folio.linked.data.service.validation.LdValidator;
+import org.folio.linked.data.service.validation.validators.LccnPatternValidator;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ValidatorsConfiguration {
```

Comment:
**Guys, why are we going to take a step back from the most modern approach to validation based on [Jakarta Bean Validation 3.0](https://beanvalidation.org/)?**

Already existed examples are: 
org.folio.linked.data.validation.PrimaryTitleConstraint
org.folio.linked.data.validation.entity.PrimaryTitleEntityValidator
org.folio.linked.data.validation.dto.PrimaryTitleDtoValidator

**Why are we going to manually build a chain of validators, manually divide them into groups for create/update, etc?**

All of that and even more is possible to implement with Jakarta Bean Validation, it's very agile and it brings less manual work and lower coupling.

Few examples with our needs:
1) Configurable enabled/disabled validations: We could make a validator a Spring bean and inject some validation configuration provider into it
2) Group validations by create/update/etc operations: [Validator groups](https://medium.com/javarevisited/javax-validation-and-groups-acd9a1fb1091) does it for us
3) Gathering a list of validation violations: provided out of the box